src/wasmtime/src/commands/run.rs

```diff
@@ -207,10 +207,10 @@ impl RunCommand {
                 // exit the cage
                 lind_syscall_api(
                     1,
-                    30 as u32,
+                    30,
```

Comment:
if we're calling syscalls directly like this this should be at least a comment saying what syscall but more appropriately we should have some defines here